integration/container/exec_test.go

```diff
@@ -17,6 +17,7 @@ import (
 
 // TestExecWithCloseStdin adds case for moby#37870 issue.
 func TestExecWithCloseStdin(t *testing.T) {
+	skip.If(t, testEnv.OSType == "windows", "FIXME. Hang with containerd")
```

Comment:
This rings a bell for me, but I can't track it down right now. There was some issue or discussion about stdin/stdout pipes states between containerd and the runtime shim from hcsshim (related to reconnecting after containerd restart, I think?), which I thought was still unresolved last time I saw it, but it was only something I noticed in-passing, not an area I was working on.

Reply:
@TBBle It looks to be that if I run container with same command than `TestCommitAfterContainerIsDone` is using and close it with CTRL + C it closes correctly:
![image](https://user-images.githubusercontent.com/6213926/119151508-24124e80-ba58-11eb-81a7-759c15542258.png)

But if I instead of hit enter couple of times and then CTRL + C is not able exit from container anymore:
![image](https://user-images.githubusercontent.com/6213926/119151773-620f7280-ba58-11eb-83af-7d4bb718be41.png)

Any idea what might causes this one? Looks that quite many of those problematic tests are actually related to stdin issues.